app/lists.rb

```diff
@@ -1,43 +1,33 @@
 module Listing
-  extend Discordrb::Commands::CommandContainer
-  
-  command :WingInitilise, description: "Initilise the array. In case of derps." do |event|
+	extend Discordrb::Commands::CommandContainer
+	
 	wing_list = Array.new
-  end
-
-  command :WingList, description: "Lists those who are requesting wing." do |event| #Simple, return the text file's contents. May consider making this a array. Also, admin lock this.
-    
-  end
-  
-  command :WingRemove, description: "Remove a person from the wing queue", min_args: 1, max_args: 1 do |event| #Remove someone who is on the wing list. Defo admin lock.
+	#The array that is wing_list is created.
 	
-  end
-
-  command :WingMe, description: "Need a wing? &WingMe" #Adds the person who used the command to the wing list.
-	to_wing = user.name # The user who typed &WingMe
-	wing_list.push(to_wing) #add them to the array-list
-	event.respond "Added to the Wing request queue."
-  end
+	command :winglist, description: "Lists those who are requesting wing." do |event|
+	#Simple, return the array's contents. Also, admin lock this.
```

Comment:
Can we just *give* them a wingcaptain role manually? cos i have no idea how to do that right now. I'll look it up though.